tests/test_other.py

```diff
@@ -1114,10 +1114,10 @@ def test_link_group_bitcode(self):
 }
 ''')
 
-    self.run_process([EMCC, '-o', '1.o', '-c', '1.c'])
-    self.run_process([EMCC, '-o', '2.o', '-c', '2.c'])
+    self.run_process([EMCC, '-flto', '-o', '1.o', '-c', '1.c'])
+    self.run_process([EMCC, '-flto', '-o', '2.o', '-c', '2.c'])
     self.run_process([EMAR, 'crs', '2.a', '2.o'])
-    self.run_process([EMCC, '-r', '-o', 'out.o', '-Wl,--start-group', '2.a', '1.o', '-Wl,--end-group'])
+    self.run_process([EMCC, '-r', '-flto', '-o', 'out.o', '-Wl,--start-group', '2.a', '1.o', '-Wl,--end-group'])
     self.run_process([EMCC, 'out.o'])
```

Comment:
I'm having trouble following this PR. Probably missing something basic. The title says to deprecate bitcode linking, but it looks like this test change makes the test use bitcode more than it did before?

Reply:
Sorry didn't add this paragraph from the commit message to the PR description:

```
Also, restore some minimal testing of this feature which had been
completely removed.
```

Does that clarify?  Basically we had no testing a all for this feature.. so this change simultaneously restores testing while deprecating the features.

Reply:
Ah, thanks! Now I am starting to see.